src/rendezvous_server.rs

```diff
@@ -40,7 +40,7 @@ const ADDR_127: IpAddr = IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1));
 
 #[derive(Clone, Debug)]
 enum Data {
-    Msg(RendezvousMessage, SocketAddr),
+    Msg(Box<RendezvousMessage>, SocketAddr),
```

Comment:
Just wondering, box is more efficient here?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant  
IMO when alloc enum in memory, the size = tag+max variant, so better to keep variants have less size differences